src/mappings/util.ts

```diff
@@ -121,15 +121,15 @@ export const getFirstValueFromJson = (item: Codec): string => {
 };
 
 export const getTextValue = (item: Codec): string => {
-  return item?.toString().trim().length === 0 ? null : item.toString().trim();
+  return item?.toString().trim().length > 0 ? item.toString().trim() : null;
 };
 
 export const getNumberValue = (item: Codec): number => {
   return Number(getTextValue(item));
 };
 
 export const getDateValue = (item: Codec): Date => {
-  return item.toString().trim().length === 0 ? null : new Date(Number(item.toString()));
+  return item.toString().trim().length > 0 ? new Date(Number(item.toString())) : null;
```

Comment:
This was fine, but I changed it so it would look the same, and if it were changed it wouldn't be the same bug as getTextValue had.